app/Modules/Auth/Resources/Lang/es/main.php

```diff
@@ -0,0 +1,52 @@
+<?php 
+
+return array(
+
+    /*
+    |--------------------------------------------------------------------------
+    | Module Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | This file keeps the language lines of the related module.
+    |
+    */
+
+    /*
+     * Logout
+     */
+    'logged_out'        => '¡Desconectado!',
+
+    /*
+     * Registration
+     */
+    'register'          => 'Registro',
+    'registered'        => '¡Registrado correctamente!',
+    'steam_registered'  => '¡Tu registro via Steam ha sido correcto! Si también quieres habilitar tu inicio de sesión via email, por favor, solicita una contraseña.',
+    'username_taken'    => 'Este nombre de usuario ya existe.',
+    'password_length'   => 'Tu contraseña debe tener al menos 6 caracteres de longitud.',
+    'register_twice'    => 'Ya estás registrado.',
+
+    /*
+     * Restore password
+     */
+    'password_reset'    => 'Recupera tu contraseña',
+    'email_gen_pw'      => 'Hemos enviado un email a tu dirección de correo electrónico. Sigue las instrucciones para generar una nueva contraseña.',
+    'email_invalid'     => 'No se ha encontrado ningún usuario con este correo eletrónico.',
+    'code_invalid'      => 'El código no coincide.',
+    'new_pw'            => 'Nueva contraseña',
+    'email_new_pw'      => 'Te hemos enviado a tu correo electrónico tu nueva contraseña.',
+    'pw_link'           => 'Por favor, haz click en el siguiente enlace para generar una nueva contraseña para :0.',
+    'email_ignore'      => 'Si no quieres generar una nueva contraseña, por favor, ignora este correo eletrónico.',
+    'pw_generated'      => 'Se ha generado una nueva contraseña para :0.',
+
+    /* 
+     * Login 
+     */
+    'login'             => 'Login',
```

Comment:
Better 'Inicar Sesión' maybe?

Reply:
Hey! You right! I'm so used to reading "login" that I overlooked at! Thanks!